README.md

```diff
@@ -21,29 +21,62 @@ oc new-app -f https://raw.githubusercontent.com/3scale/apicast/v2/openshift/apic
 
 ## Docker
 
-You can download a ready to use docker image from our repository:
+You can download a ready to use Docker image from our repository:
 
 ```
 docker pull quay.io/3scale/apicast:v2
 ```
 
-The 3scale gateway image requires two ENV variables:
+The 3scale gateway image requires one of the two environment variables:
 
 * **THREESCALE_PORTAL_ENDPOINT**
 
 URI that includes your password and portal endpoint in following format: `schema://access-token@domain`.
-Example: https://provider-key@test-admin.3scale.net
+Example: https://provider-key@account-admin.3scale.net
+
+When `THREESCALE_PORTAL_ENDPOINT` environment variable is provided, the gateway will download the configuration from the 3scale on initializing. The configuration includes all the settings provided on the Integration page of the API(s).
+
+```
+docker run --name apicast --rm -p 8080:8080 -e THREESCALE_PORTAL_ENDPOINT=https://provider-key@account-admin.3scale.net quay.io/3scale/gateway:v2
+```
 
 * **THREESCALE_CONFIG_FILE**
 
-Path to saved JSON file with configuration for the gateway. Has to be injected to the docker image as read only volume.
+Path to saved JSON file with configuration for the gateway. The file has to be injected to the docker image as read only volume, and the path should indicate where the volume is mounted, i.e. path local to the docker container.
+
+```
+docker run --name apicast --rm -p 8080:8080 -v $(pwd)/config.json:/opt/app/config.json -e THREESCALE_CONFIG_FILE=/opt/app/config.json quay.io/3scale/gateway:v2
+```
+
+In this example `config.json` is located in the same directory where the `docker` command is executed, and it is mounted as a volume at `/opt/app/config.json`
 
-#### Docker command
+The JSON file needs to follow the the [schema](schema.json), see an example file with the fields that are used by the gateway [here](examples/configuration/example-config.json).
+
+In some 3scale plans it is possible to create multiple API services (see an example of the configuration file [here](examples/configuration/multiservice.json)). The _optional_ **APICAST_SERVICES** environment variable allows filtering the list of services, so that the gateway only includes the services explicitely specified, the value of the variable should be a comma-separated list of the service IDs. This setting is useful when you have many services configured on 3scale, but you want to expose just a subset of them in the gateway.
```

Comment:
I'd rather do: see **an example configuration file**. as a link. That is what I learned in my SEO days, that using `here` is not great.